### PR TITLE
fix: edit getMMDDYYYYfromISO function in utils.js

### PR DESCRIPTION
**Pull request by lee3665**
Add the case when the type of date parameter is null (when the date input is empty)

### CODE REVIEW

**Review by gonzalezlrjesus — COMMENTED** [review]
everything's okay! thanks @lee3665

**Comment by gonzalezlrjesus** [comment]
This PR resolve the error generated by date fields could be null

File: src/utils/utils.js [hunk 1]
```diff
@@ -7,14 +7,18 @@ export function getISOfromDatepicker(date) {
...
+	} else {
+		return date.slice(0, 10);
+	}
 }
 
 //date is of type iso string
 export function getAge(date) {
 	let parsed = Date.parse(date)
 	let diff = Date.now() - parsed
-	let ageDate = new Date(diff)
```
Comment (sjaymoon15) [inline src/utils/utils.js:17]: Just noticed this potential error probably caused by accident. 
We would need to change `ageDaete` back to `ageDate`.  @lee3665 
ageDate is used in the return statement. 
fyi @gonzalezlrjesus

Comment (gonzalezlrjesus) [reply src/utils/utils.js:17]: [Commit](https://github.com/Raise-the-Voices/marturia/commit/a42fb09d028befdf5fba55f00c9b6370547977fd#commitcomment-44091060) @sjaymoon15

**Pull request merged**